omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/VisitResource1_9.java

```diff
@@ -309,10 +309,11 @@ public SimpleObject search(RequestContext context) throws ResponseException {
 	
 	private SimpleObject getVisits(RequestContext context, String patientParameter, String includeInactiveParameter,
 	        Date minStartDate) {
+		boolean includeAll = Boolean.parseBoolean(context.getParameter("includeAll"));
```

Comment:
Validate the boolean input from the request.

Reply:
Check this out: [validate Boolean](https://github.com/openmrs/openmrs-module-webservices.rest/blob/639ad0273ee0c086cd823e3c118e657ec4f0b09e/omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/ObsResource1_8.java#L438)

Reply:
@adamsdenniskariuki using the example you gave looks a little bit out of context. The getParameter returns a string, or a null, whatever the case, the parseBoolean will return true or false. I do not think having 8 lines of code to validate that would be necessary.

Reply:
True

Reply:
Thank you